app/routines/admin/search_security_log.rb

```diff
@@ -0,0 +1,153 @@
+# Routine for searching the Security Log by admins
+#
+# Caller provides a query and some options.  The query follows the rules of
+# https://github.com/bruce/keyword_search , e.g.:
+#
+#   "user:jps,richb" --> returns the security log for the "jps" and "richb" users
+#   "application:Tutor" --> returns the security log for the Tutor application
```

Comment:
Could just support one of three forms:

`after X`
`before X`
`X to Y`

and pass `X` and `Y` to Chronic.  Doesn't have to be perfect since it is just for Greg :-)